src/stories/HeatmapVis.stories.tsx

```diff
@@ -103,6 +103,19 @@ LiveDataWithoutLoader.args = {
   showLoader: false,
 };
 
+export const CustomCoordinates = Template.bind({});
+
+CustomCoordinates.args = {
+  dataArray,
+  domain,
+  abscissas: Array(dataArray.shape[1] + 1)
+    .fill(0)
+    .map((x, i) => 100 + 10 * i),
+  ordinates: Array(dataArray.shape[0] + 1)
+    .fill(0)
+    .map((x, i) => -5 + 0.5 * i),
```

Comment:
`lodash`'s `range` function accepts a step https://lodash.com/docs/4.17.15#range

Reply:
True, but that means I have to compute the `end`  to have the number of elements I need :smile:

Reply:
Well, you do anyway when you initialize the array, don't you? :D

Reply:
Ah no, because the end has to be computed with the step... Well that sucks.

Reply:
Shame there is no equivalent to [linspace](https://numpy.org/doc/stable/reference/generated/numpy.linspace.html)